public/javascripts/ng-controllers.js

```diff
@@ -97,10 +95,63 @@ springInitiative.controller('studentController', function($scope,  $rootScope, $
       });
   }
 
+  $scope.showData = function(currentStudent){
+    $http.get('/api/student/data/'+ currentStudent._id)
+      .success(function(data){
+        $scope.attendance = data.attendanceList;
+        $scope.dates = data.datesList;
+        $scope.stars = data.starsList;
+
+        console.log($scope.attendance, $scope.attendance.sort())
+
+        var  count = {}; 
+        $scope.attendance.forEach(function(i) { count[i] = (count[i]||0)+1;  });
```

Comment:
It basically creates an object that turns ['Present', 'Present', 'Absent'] into {Present:2, Absent:1} so that we can count the amount of each